### PR TITLE
Feature/code tidy up rename related content

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
https://trello.com/c/0RwVsdm2/980-code-tidy-up-rename-relatedcontent

> If this is an issue, do we have steps to reproduce?
n/a
### Intent

> What changes are introduced by this PR that correspond to the above card?

- rename hubRelatedContent component to hubContent to enable generic reuse of the component where content being displayed isn't 'related content'
- Add object key rename util and supporting tests

> Would this PR benefit from screenshots?
no
### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a
> Are there any steps required when merging/deploying this PR?
n/a
### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

**Review by leonk — APPROVED** [review]
LGTM

File: server/routes/tags.js [hunk 1]
```diff
@@ -32,11 +33,13 @@ const createTagRouter = ({ cmsService }) => {
         detailsType: 'small',
       };
 
-      const data = await cmsService.getTag(
+      let data = await cmsService.getTag(
         req.session.establishmentName,
         parseInt(id, 10),
       );
 
+      data = renameKey(data, 'relatedContent', 'hubContent');
```
[the remark this answers is not in the document]
Comment (jonhindle) [reply server/routes/tags.js:41]: Also naming a parameter the same as a component is dangerous. The component named _hubContent_ seems ok, but the data should be something different e.g. _hubContentData_?

File: server/views/pages/collections.html [hunk 2]
```diff
@@ -31,11 +31,10 @@ <h1 id="title" class="govuk-heading-xl">{{ title }}</h1>
...
 
 {% block content %}
-
-  {% if data.relatedContent|length %}
-    {{ hubRelatedContent({
+  {% if data.hubContent|length %}
+    {{ hubContent({
       id: tagId,
-      data: data.relatedContent,
+      data: data.hubContent,
```
Comment (jonhindle) [inline server/views/pages/collections.html:37]: Component and data having the same name is potentially confusing

File: server/routes/tags.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (BarryGeeMOJ) [reply server/routes/tags.js]: refactored.

File: server/views/pages/collections.html (hunk 2 above)
Comment (BarryGeeMOJ) [reply server/views/pages/collections.html]: refactored

**Pull request merged**